### DISCUSSION TITLE
Options for AM/PM or 24HR on overlay

### DISCUSSION THREAD

**Discussion by itsalex97** [zmanowar/random-event-analytics#15] [Ideas]
Currently It displays the date and time that the random event spawned, but it can be confusing as it will display the time in 12 hours regardless of whether it's AM or PM and gives no indication of whether it was AM or PM.
A proposed change would be to default add AM/PM to the overlay, with the option in the settings of changing to 24 hour time if preferred.
![image](https://github.com/zmanowar/random-event-analytics/assets/5008283/dd3aca8a-da96-47c0-a72d-6fd494c86f24)

**Comment by zmanowar** [comment]
Good idea! This is solved and configurable in the most recent commits. https://github.com/zmanowar/random-event-analytics/compare/e88e1cc24e765ea25b12601383d7e825c39bdf8f...master -- also made the hover tooltip respect it as well. 
![image](https://github.com/zmanowar/random-event-analytics/assets/39996391/101b721e-29dd-4d61-a075-619d6f44ed80)

Edit: Will put up a plugin-hub PR by the end of the week (or sooner if there are other bugs/changes).
